### PR TITLE
Ability to recover from webGL context loss

### LINKED ISSUE #6653: Recover from webGL context loss
Handle the `webglcontextrestored` event to recover from a context loss. Needs investigation to find out how much threejs can do for us here and how much of our own rendering logic has to be touched.

### PR DESCRIPTION
**Pull request by philippotto**
### URL of deployed dev instance (used for testing):
- https://restorefromcontextloss.webknossos.xyz

### Steps to test:
- open a dataset or an annotation
- open the console
- execute `window.testContextLoss()`
- wait for a few seconds (the recovery event is dispatched after 2.5 seconds which then triggers the actual recovery mechanism)

### Issues:
- fixes #6653 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/libs/UpdatableTexture.ts [hunk 1]
```diff
@@ -152,7 +152,6 @@ class UpdatableTexture extends THREE.Texture {
       src,
     );
     this.gl.bindTexture(this.gl.TEXTURE_2D, activeTexture);
-    this.image = null;
```
Comment (philippotto) [inline frontend/javascripts/libs/UpdatableTexture.ts:155]: This resource is necessary when re-uploading textures to the GPU after a context recovery. I compared the heap memory consumption between this branch and the master and could not find a noticeable difference.

File: frontend/javascripts/oxalis/controller/renderer.ts [hunk 2]
```diff
@@ -20,10 +17,24 @@ function getRenderer() {
...
+  const ext = renderer.getContext().getExtension("WEBGL_lose_context");
+  if (ext == null) {
+    return;
+  }
+  ext.loseContext();
+  setTimeout(() => ext.restoreContext(), 2500);
+}
+
+// @ts-ignore
+window.testContextLoss = testContextLoss;
```
Comment (philippotto) [inline frontend/javascripts/oxalis/controller/renderer.ts:37]: I'll remove this after testing.

File: .circleci/not-on-master.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
-if [ "${CIRCLE_BRANCH}" == "master" ]; then
+# if [ "${CIRCLE_BRANCH}" == "master" ]; then
```
Comment (daniel-wer) [inline .circleci/not-on-master.sh:4]: Reminder to comment this back in

**Review by daniel-wer — APPROVED** [review]
Nice solution, works very well :+1: 

After adding a changelog entry and reverting the temporary changes, this should be good to go.

**Pull request merged**